theory/DDrppi/countpairs_rp_pi_impl.c.src

```diff
@@ -334,7 +334,8 @@ int countpairs_rp_pi_DOUBLE(const int64_t ND1, DOUBLE *X1, DOUBLE *Y1, DOUBLE *Z
                                                                          xdiff, ydiff, zdiff,
                                                                          -1.0, rpmax, pimax,/*max_3D_sep, max_2D_sep, max_1D_sep*/
                                                                          options->enable_min_sep_opt,
-                                                                         autocorr, options->periodic);
+                                                                         autocorr,
+                                                                         options->periodic, options->periodic, options->periodic);
```

Comment:
Should this be periodic_x/y/z?

Reply:
Yes, once we finalize the DD design and port the feature to the other modules.